packages/shared/lib/core/wallet/utils/transactions/getSenderAddressFromInputs.ts

```diff
@@ -14,14 +14,18 @@ export function getSenderAddressFromInputs(inputs: IWrappedOutput[]): string | u
         const { output, metadata } = input
         const unlockConditions = (output as CommonOutput)?.unlockConditions
 
-        const spentDate = metadata?.milestoneTimestampSpent
+        // TODO(2.0): modify this
```

Comment:
perhaps we should create an issue for this and add it to the backlog, just to make sure we don't forget about it, wdyt? :pray:

Reply:
Modified! ✅